CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 ### Added
 
 - Added ConVar to toggle double-click buying
+- Added a new vgui system
+  - the F1 menu is completely overhauled
```

Comment:
why is this in `Added` and not in `Changed` ?

Reply:
I thought about this as well ... but since the F1 screen is redone from scratch I think it fits better to added, but I dunno